doc/nrf/protocols/bt/bt_mesh/dfu_over_ble.rst

```diff
@@ -16,6 +16,11 @@ This happens if the device is not advertising the SMP service UUID and the filte
 The device can still be discovered through a service discovery, for example using the `nRF Connect for Mobile`_ app.
 See `Discovering Bluetooth Mesh devices in nRF Connect Device Manager`_ for more details.
 
+Before performing a FOTA update, make sure the option to erase application settings is disabled, otherwise the device will be unprovisined after the update.
```

Comment:
```suggestion
Before performing a FOTA update, make sure that you have disabled the option to erase application settings.
Otherwise, the device will be unprovisioned after the update.
```